app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <debug-overrides>
+        <trust-anchors>
+            <certificates src="user" />
```

Comment:
do you mind adding comment that this certificates are special for Charles?

Reply:
this is not only for Charles